themes/bootstrap3/templates/RecordDriver/SolrDefault/data-summary.phtml

```diff
@@ -0,0 +1,13 @@
+<? 
+    $summaries = $this->driver->getSummary(); 
+    foreach ($summaries as $summary):
+        ?><?=$summary ?><br /><?
+    endforeach;
+    $isbn = $this->driver->getCleanISBN();
+    $summaries = $this->summaries($isbn);
+    foreach ($summaries as $provider => $list) {
+        foreach ($list as $summary) {
+            ?><?=$summary ?><br /><?
```

Comment:
Do we also need to escape this summary, or is the API giving us HTML?

Reply:
Added. I'll push these changes up in a minute.